Assets/Puppet/PuppetReleaseHandler.cs

```diff
@@ -7,20 +7,22 @@
 // Very simple release handler.
 public class PuppetReleaseHandler : MonoBehaviour
 {
-    private static ReleaseDetails _releaseDetails = null;
+    private static ReleaseDetails _releaseDetails;
     private static readonly object _releaseLock = new object();
+    public static bool IsDialogCustom;
```

Comment:
```suggestion
    public static bool IsCustomDialog;
```

Reply:
That's what I called it at first. But it's not grammatically correct and sounds weird.